README.md

```diff
@@ -34,10 +34,116 @@ or `git clone https://github.com/suhailshergill/extensible-effects.git`
 2. start `stack ghci` or `cabal repl`
 3. import some library modules as described in this section
 
-*examples are a work in progress and there will be some Quickstart module to go
-along the guide here*
+### Examples
 
-*examples...*
+In this section, a lot of examples are given. For technical details, see the
+next section.
+
+Note that most examples given here are very small. For them,
+using `Eff` monad is more complicated.
+The power of extensible effects lie in the fact that these comutations can be
+used to construct much more complicated programs by composing the little pieces
+shown here.
+
+All examples from this module can be found and tested in the module
+`Control.Eff.Quickstart`.
```

Comment:
@power-fungus shouldn't we have `Control.Eff.Quickstart.Test` as well with the tests?

Reply:
I don't know if this is necessary. the correctness of the examples are checked with doctest and the Test-modules cannot be imported in ghci from another module as far as I know.